modules/search-service-api/src/main/java/org/opencastproject/search/api/SearchQuery.java

```diff
@@ -48,7 +48,7 @@
 
   public enum Sort {
     DATE_CREATED,
-    DATE_PUBLISHED,
+    DATE_MODIFIED,
```

Comment:
Oof okay. Can we break the search API in a new major version? We can of course add a simple condition for the old name (rewriting it to the new one) to preserve backwards compatibility. It's not great but in the broad scheme of things, this tiny piece of code is probably worth it to avoid breaking everything?

Reply:
Not sure about the rules regarding Search. :thinking: @lkiesow?

As for adding a small compatibility patch: I'm personally not a big fan of stuff like that but I guess in this case it's simple enough? :shrug: But in the long run it would be great if we could get rid of it. :thinking: